### PR TITLE
Updates PR template for new process

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)

Updates this very PR template
#### What tests does this PR have?

NONE
#### How can this be tested?

CAN'T
#### What domains are affected?

NONE
#### What gif best describes how you feel about this work?

DOG

---
- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.
- Thought about whether this PR needs additional reviewers and requested if necessary.

### CODE REVIEW

File: pr-template.md [hunk 1]
```diff
@@ -1,35 +1,30 @@
...
-- [ ] :+1:
-
-**Review 2**
-- [ ] :+1:
+---
 
-**Review 3**
+**Reviewer**
 - [ ] :+1:
+- [ ] This PR need any additional reviewers!
```
Comment (msaspence) [inline pr-template.md]: Can we rephrase this to the negative. "I don't think this PR needs any additional reviewers"

This way indicating that it does need reviewers doesn't move the progress bar in the PR list view along.

File: pr-template.md [hunk 2]
```diff
@@ -1,35 +1,30 @@
...
 
 By adding a +1 you are confirming you have...
```
Comment (msaspence) [inline pr-template.md:30]: Does this and the check on line 16 add any real value. I think it just adds noise to the description and people see it too often to actually shock them into changing their behaviour. 

It feels a little like the immigrations question "Do you or have you had any links to Nazis".

File: pr-template.md (hunk 1 above)
Comment (theninj4) [inline pr-template.md]: :+1:

File: pr-template.md (hunk 2 above)
Comment (theninj4) [inline pr-template.md:30]: :+1:
